### ISSUE TITLE
DNN library is not found. File "/usr/local/lib/python3.7/dist-packages/keras/backend.py", line 5366, in relu       x = tf.nn.relu(x) Node: 'sequential/conv2d/Relu'  	 [[{{node sequential/conv2d/Relu}}]] [Op:__inference_train_function_13695]

### ISSUE THREAD

**Issue by TalalAhmed311** [googlecolab/colabtools#3073]
I am getting this error in colab notebook don't know why. During model building i also restart the runtime but still the error is coming.

I am using Brave browser.

Here is my File:
https://colab.research.google.com/drive/1NcMPpUpwWQHVz7XlEGDt8v-NXAwafIY1?usp=sharing

  model = Sequential()
  model.add(Conv2D(16, (3,3), activation='relu', input_shape=(1491, 257,1)))
  model.add(Conv2D(16, (3,3), activation='relu'))
  model.add(Flatten())
  model.add(Dense(128, activation='relu'))
  model.add(Dense(1, activation='sigmoid'))
model.compile('Adam', loss='binary_crossentropy',metrics=['accuracy'])

**Comment by mayankmalik-colab** [comment]
I can see in the notebook that the tensorflow version is `2.10.0`, which must have been upgraded manually. This can make our cudnn version incompatible.

Can you try the following:
`!apt install --allow-change-held-packages libcudnn8=8.1.0.77-1+cuda11.2` .

Note: It would require restarting your runtime.
